plugins/node/opentelemetry-instrumentation-pg/src/utils.ts

```diff
@@ -158,7 +155,7 @@ export function handleConfigQuery(
   ) {
     span.setAttribute(
       AttributeNames.PG_VALUES,
-      arrayStringifyHelper(queryConfig.values)
+      JSON.stringify(queryConfig.values)
```

Comment:
I recommend wrapping this function with try-catch, as `JSON.stringify` can throw for cyclic objects, and `queryConfig.values` is a user supplied argument for which we cannot assume anything.